doc/perception/experiments/CLRerNet - lanedetection/Dockerfile

```
@@ -0,0 +1,280 @@
+# avoid python warnings about missing bin directory in PATH
+# (as we're not running as root, pip installs into ~/.local/bin)
+ENV PATH=$PATH:/home/$USERNAME/.local/bin
+
+# install pip packages
+RUN python -m pip install pip --upgrade \
+    && python -m pip install simple_pid pygame transformations roslibpy lxml black
+
+# install the scenario runner from GitHub leaderboard-2.0 branch
+ENV CARLA_ROOT=/opt/carla
+ENV SCENARIO_RUNNER_ROOT=/opt/scenario_runner
+RUN sudo mkdir $SCENARIO_RUNNER_ROOT && sudo chown $USERNAME:$USERNAME $SCENARIO_RUNNER_ROOT
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)